ui/src/app/pages/analysis/util.ts

```diff
@@ -41,6 +42,14 @@ export function appendRScriptSuffix(filename: string) {
   return filename;
 }
 
+export function appendSasSuffix(filename: string) {
+  if (filename && !filename.endsWith(SAS_FILE_EXT)) {
+    return filename + SAS_FILE_EXT;
+  }
+
+  return filename;
+}
+
 export function appendAnalysisFileSuffixByOldName(
```

Comment:
This function does exactly the same thing as the API's `appendFileExtensionIfNotExist()` - it would be nice to standardize on the same function name and the same parameter name in this case.

I am not arguing for exposing the API function itself as an endpoint.  I think that would be too heavyweight for this simple task.